ng-ncc-app/src/app/API/HackneyAPI/hackney-api.service.spec.ts

```diff
@@ -1,15 +1,20 @@
 import { TestBed, inject } from '@angular/core/testing';
+import { HttpClient } from '@angular/common/http';
 
 import { HackneyAPIService } from './hackney-api.service';
 
+let httpClientSpy: { get: jasmine.Spy };
+let HackneyAPI: HackneyAPIService;
+
 describe('HackneyAPIService', () => {
-  beforeEach(() => {
-    TestBed.configureTestingModule({
-      providers: [HackneyAPIService]
+    beforeEach(() => {
+        // from the Angular tutorial: https://angular.io/guide/testing#testing-http-services
+        httpClientSpy = jasmine.createSpyObj('HttpClient', ['get']);
+        HackneyAPI = new HackneyAPIService(<any>httpClientSpy);
+    });
+
+    it('should be created', () => {
+        expect(HackneyAPI).toBeTruthy();
     });
-  });
 
-  it('should be created', inject([HackneyAPIService], (service: HackneyAPIService) => {
-    expect(service).toBeTruthy();
-  }));
 });
```

Comment:
Service unit tests are being prepared to use spies of dependencies instead of the dependencies themselves. In this case the HttpClient is being mocked, so no actual HTTP requests will be made.